src/sentry/static/sentry/app/components/organizations/headerItemPosition.jsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'react-emotion';
+import {AutoCompleteRoot} from 'app/components/dropdownAutoCompleteMenu';
+import {TimeRangeRoot} from 'app/components/organizations/timeRangeSelector/index.jsx';
+
+const HeaderItemPosition = styled.div`
+  display: flex;
+  flex: 1;
+  max-width: 450px;
+  height: 100%;
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  ${AutoCompleteRoot}, ${TimeRangeRoot} {
```

Comment:
this is maybe ok in the short term but worrying as a pattern since it will have a lot of the same issues as traditional css scoping.